gulpfile.js

```diff
@@ -16,46 +16,15 @@ const appPaths = {
   src: 'src',
   dist: 'dist'
 }
-
-const modules = {
-  common: ['d3', 'colorbrewer', 'font-awesome', 'require'],
-
-  other: ['common'],
-  tree: ['common'],
-
-  chart: ['other'],
-  graph: ['dagre', 'other'],
-
-  c3chart: ['c3', 'chart'],
-  google: ['chart' /* Google Visualization */],
-  map: ['graph', 'topojson' /* , Google Map */],
-  
-  layout: ['c3chart'],
-  marshaller: ['chart', 'graph', 'layout']
-}
-
-// Helpers
-function getModuleDeps(module) {
-  if (!modules[module]) return []
-
-  return modules[module]
-    .reduce(function (deps, dep) { 
-      return deps.concat(dep).concat(getModuleDeps(dep)) 
-    }, [])
-    .filter(function (dep, index, col) { 
-      return col.indexOf(dep) === index 
-    })
-}
+const libs = ["d3", "c3", "colorbrewer", "dagre", "topojson", "d3.layout.cloud", "font-awesome"];
 
 function buildModule(module, cb) {
   gutil.log('Building ' + module + '...')
 
-  dir.files(appPaths.src, function (err, files) {
```

Comment:
dir.files caused issues on windows machines due to "\" versus "/" in file paths.